src/coreclr/gc/gcpriv.h

```diff
@@ -4475,6 +4479,9 @@ class gc_heap
     // at the beginning of a BGC and the PM triggered full GCs
     // fall into this case.
     PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t suspended_start_time;
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t suspended_end_time;
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t h0_gc_start_time;
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t change_heap_count_time;
```

Comment:
this is actually not diag only. I think I added them for diag only at first but it was used for tuning. so we should move it to be after this line
`PER_HEAP_ISOLATED_FIELD_MAINTAINED uint64_t last_suspended_end_time;`
and change it to `PER_HEAP_ISOLATED_FIELD_MAINTAINED`